### PR TITLE
Documenting extension release process

### PR DESCRIPTION
**Pull request by Coldarn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +40,9 @@ For a release build, you can also run `yarn release` for a much smaller applicat
    - If you added the extension as a `Case home page`:
       1. Navigate to the Case Home page.
       1. Click the extension tab in the navigation pane.
+
+## How to deploy it
```
Comment (sereana-fti) [inline README.md]: Is this just for the release build? If so, you might want to clarify. E.g. "How to deploy it to production"

**Comment by Coldarn** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (Coldarn) [reply README.md]: Ah, good point.

Comment (Coldarn) [reply README.md]: I pushed this change up out-of-band. You should see it now on the front page of this repository.

**Pull request merged**